packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -4,207 +4,411 @@
  * @format
  */
 
-import path from 'path';
-import {randomBytes} from 'crypto';
 import * as appInsights from 'applicationinsights';
-import {execSync} from 'child_process';
-import {CodedError} from './CodedError';
 
+import * as basePropUtils from './utils/basePropUtils';
+import * as versionUtils from './utils/versionUtils';
+import * as errorUtils from './utils/errorUtils';
+import * as projectUtils from './utils/projectUtils';
+
+export interface TelemetryOptions {
+  setupString: string;
+  preserveErrorMessages: boolean;
+}
+
+export interface CommandStartInfo {
+  commandName: string;
+  args: Record<string, any>;
+  options: Record<string, any>;
+  defaultOptions: Record<string, any>;
+}
+
+export interface CommandEndInfo {
+  resultCode: errorUtils.CodedErrorType;
+}
+
+interface CommandInfo {
+  startTime?: number;
+  endTime?: number;
+  startInfo?: CommandStartInfo;
+  endInfo?: CommandEndInfo;
+}
+
+const DefaultSetupString = '795006ca-cf54-40ee-8bc6-03deb91401c3';
+
+export const NpmPackagesWeTrack: string[] = [
+  '@react-native-community/cli',
+  '@react-native-windows/cli',
+  '@react-native-windows/telemetry',
+  'react',
+  'react-native',
+  'react-native-windows',
+  'react-native-windows-init',
```

Comment:
I think it can still work because I'm using `require` to find the package.json, but unfortunately I can't test this until I get it checked in so I can call `react-native-windows-init` without any workarounds like yarn link.